libs/localization/src/lib/useApplicationNamespaces.tsx

```diff
@@ -1,12 +1,16 @@
 import {
   ApplicationTypes,
-  ApplicationTranslations,
+  ApplicationConfigurations,
 } from '@island.is/application/core'
 
 import { useNamespaces } from './useNamespaces'
 
-export const useApplicationNamespaces = (type: ApplicationTypes) => {
-  const array = ['application.system', ApplicationTranslations?.[type] ?? null]
+export const useApplicationNamespaces = (type?: ApplicationTypes) => {
+  const array = [
+    'application.system',
+    type ? ApplicationConfigurations?.[type].translation ?? null : null,
```

Comment:
In theory this could end up as `undefined` if `ApplicationConfigurations?.[type]` is not defined.